Hearthstone Deck Tracker/Windows/DeckScreenshotWindow.xaml.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using Hearthstone_Deck_Tracker.Hearthstone;
+using Hearthstone_Deck_Tracker.Utility.Extensions;
+using static System.Windows.Visibility;
+
+namespace Hearthstone_Deck_Tracker.Windows
+{
+	public partial class DeckScreenshotWindow
+	{
+		private bool _showDeckOnly;
+		private const int EXTRAS_HEIGHT = 40 + 24 + 34 + 24;
+
+		public DeckScreenshotWindow(Deck deck, bool deckOnly = false)
+		{
+			InitializeComponent();
+
+			_showDeckOnly = deckOnly;
+
+			ListViewPlayer.Update(deck.Cards.ToSortedCardList(), true);
+
+			if(_showDeckOnly)
+			{
+				DeckTitleContainer.Visibility = Collapsed;
+				DeckFormatPanel.Visibility = Collapsed;
+				SetDustPanel.Visibility = Collapsed;
+				BrandContainer.Visibility = Collapsed;
+			}
+			else
+			{
+				DeckTitlePanel.Background = DeckHeaderBackground(deck.Class);
+				LblDeckTitle.Text = deck.Name;
+				LblDeckTag.Text = GetTagText(deck);
+				LblDeckFormat.Text = GetFormatText(deck);
+				LblDustCost.Text = TotalDust(deck).ToString();
+				ShowFormatIcon(deck);
+				SetIcons.Update(deck);
+			}
+
+			Height = 35 * ListViewPlayer.Items.Count + (_showDeckOnly ? 0 : EXTRAS_HEIGHT);
+		}
+
+		public double DeckMaxHeight => ActualHeight - LabelsHeight;
+
+		public double LabelsHeight => DeckTitleContainer.ActualHeight + DeckFormatPanel.ActualHeight
+			+ SetDustPanel.ActualHeight + BrandContainer.ActualHeight + 42;
+
+		public void UpdateWindowHeight()
+		{
+			this.Height = StackPanelMain.ActualHeight + 42;

```

Comment:
No need for `this`. Should be converted to an expression (`public void UpdateWindowHeight() => Height = StackPanelMain.ActualHeight + 42;`)